src/main/webapp/WEB-INF/views/customerSearchScreen.html

```diff
@@ -0,0 +1,146 @@
+                </li>
+
+                <li class="nav-item mx-2">
+                    <a class="nav-link" href="#">Application</a>
+                </li>
+
+                <li class="nav-item mx-2">
+                    <a class="nav-link" href="#">Receipt</a>
+                </li>
+
+                <li class="nav-item dropdown dmenu mx-2">
+                    <a class="nav-link dropdown-toggle" href="#" id="navbardrop3" data-toggle="dropdown">
+                        Accounting
+                    </a>
+                    <div class="dropdown-menu sm-menu">
+                        <a class="dropdown-item" href="#">Link 1</a>
+                        <a class="dropdown-item" href="#">Link 2</a>
+                        <a class="dropdown-item" href="#">Link 3</a>
+                    </div>
+                </li>
+
+                <li class="nav-item mx-2">
+                    <a class="nav-link" href="#">Customer Service</a>
+                </li>
+
+                <li class="nav-item dropdown dmenu mx-2">
+                    <a class="nav-link dropdown-toggle" href="#" id="navbardrop4" data-toggle="dropdown">
+                        Report
+                    </a>
+                    <div class="dropdown-menu sm-menu">
+                        <a class="dropdown-item" href="#">Link 1</a>
+                        <a class="dropdown-item" href="#">Link 2</a>
+                        <a class="dropdown-item" href="#">Link 3</a>
+                    </div>
+                </li>
+
+            </ul>
+            <div class="social-part">
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+                <i class="fa fa-instagram" aria-hidden="true"></i>
+            </div>
+        </div>
+    </nav>
+    <!-- NavBar Ends -->
+
+    <div class="row pt-3 pl-3 flex-column">
+        <h2 class="  display-3" style="font-size: 30px">
+            <b> Customer Service</b>
+        </h2>
+    </div>
+
+    <hr width="" color="#b3b3b3">
+
+</div>
+
+<!-- Form Container -->
+<div class="container-fluid">
+    <form>
+        <div class="row">
+            <div class="col-sm-3">
+
+                <div class="form-group">
+                    <label for="productCode" class="font-weight-bold ">Customer #</label>
+                    <input type="text" class="form-control" id="productCode">
+                </div>
+
+
+            </div>
+            <div class="col-sm-3 offset-sm-4">
+
+                <div class="form-group">
+                    <label for="productName" class="font-weight-bold ">Loan Account #</label>
+                    <input type="text" class="form-control" id="productName">
+                </div>
+
+            </div>
+        </div>
+
+        <hr width="" color="#b3b3b3">
+
+    </form>
+
+    <div class="row">
+        <div class="col-sm-8 offset-sm-9">
+            <button type="button" class="btn btn-primary">Search</button>
```

Comment:
why not add ids to the buttons? the id can be same as the name, like 'search' or 'clear'.